crates/amaru-ledger/src/governance.rs

```diff
@@ -19,6 +22,22 @@ impl DRepsSummary {
     pub fn new(db: &impl Snapshot) -> Result<Self, StoreError> {
         let mut dreps = BTreeMap::new();
 
+        let current_epoch = db.most_recent_snapshot();
```

Comment:
Got me confused for a second. We should rename `most_recent_snapshot` to `epoch`, since this is now only called on snapshots and they -- by definition -- refer to only one epoch.